libnavigation-ui/src/main/java/com/mapbox/navigation/ui/instruction/GuidanceViewImageProvider.kt

```diff
@@ -83,17 +66,20 @@ class GuidanceViewImageProvider() {
 
     /**
      * The API allows you to cancel the rendering of guidance view.
-     * @param context Context
      */
-    fun cancelRender(context: Context) {
-        if (targets.isNotEmpty()) {
-            for(target in targets) {
-                Picasso.Builder(context).build().cancelRequest(target)
-            }
-            targets.clear()
+    fun cancelRender() {
+        if (coroutineScope.isActive) {
+            coroutineScope.cancel()
```

Comment:
if you cancel a scope, it will be impossible to launch any coroutines on that scope. 
Is it expected behaviour?

as mentioned above, it's better to use ThreadController and `cancelChildren` of the scope